File: backend/src/main/java/com/coing/global/config/JacksonConfig.java
package com.coing.global.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.SerializationFeature;
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;

@Configuration
public class JacksonConfig {
	@Bean
	public ObjectMapper objectMapper() {
		return new ObjectMapper()
			.registerModule(new JavaTimeModule()) // Java 8 날짜/시간 지원 추가
			.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS); // timestamp 변환 방지
	}
}

File: backend/src/main/java/com/coing/global/config/JacksonConfig.kt
package com.coing.global.config

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.SerializationFeature
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
import com.fasterxml.jackson.module.kotlin.KotlinModule
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class JacksonConfig {

	@Bean
	fun objectMapper(): ObjectMapper {
		return ObjectMapper()
			.registerModule(KotlinModule.Builder().build())
			.registerModule(JavaTimeModule()) // Java 8 날짜/시간 지원 추가
			.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS) // timestamp 변환 방지
	}
}
